src/Core/Framework/Resources/config/packages/framework.yaml

```diff
@@ -215,14 +215,19 @@ framework:
             log_level: notice
         Shopware\Core\Content\Seo\Exception\InvalidTemplateException:
             log_level: notice
+        # @deprecated tag:v6.7.0 - Exception will be deleted
         Shopware\Core\Checkout\Document\Exception\InvalidDocumentException:
             log_level: notice
+        # @deprecated tag:v6.7.0 - Exception will be deleted
         Shopware\Core\Checkout\Document\Exception\DocumentGenerationException:
             log_level: notice
+        # @deprecated tag:v6.7.0 - Exception will be deleted
         Shopware\Core\Checkout\Document\Exception\DocumentNumberAlreadyExistsException:
             log_level: notice
+        # @deprecated tag:v6.7.0 - Exception will be deleted
         Shopware\Core\Checkout\Document\Exception\InvalidDocumentRendererException:
             log_level: notice
+        # @deprecated tag:v6.7.0 - Exception will be deleted
         Shopware\Core\Checkout\Document\Exception\InvalidFileGeneratorTypeException:
             log_level: notice
```

Comment:
Please add the respective code for the replacement into the `shopware.yaml -> shopware.logger.error_code_log_levels`